config/nova/interactions/slap_ass.json

```
@@ -0,0 +1,29 @@
+{
+	"name": "Slap ass",
+	"description": "Slap their butt.",
+	"usage": "other",
+	"category": " Petting",
```

why is there a space here before it? Should be clearer that it's sexual in nature

```suggestion
	"category": "Heavy Petting",
```
Or Sex (Foreplay) would work also
(applies to all the others too)